src/styles/helpers.scss

```diff
@@ -0,0 +1,28 @@
+$gs-gutter: 20px;
+$gs-max-columns: 16;
+$gs-column-width: 60px;
+
+@function gs-span($n-columns) {
+	@return $n-columns * $gs-column-width + $gs-gutter * ($n-columns - 1);
+}
+
+.gs-container {
+	position: relative;
+	margin: 0 auto;
+
+	@media (min-width: 740px) {
+		padding: 0 calc(50% - #{(gs-span(9) + $gs-gutter * 2) / 2});
```

Comment:
I think we simplified this at some point? seems to have been lost?

Reply:
I made the change and it seems like it is working fine now. Done in 16ee12bd58bab40ad2a19e4e4e64ab5979374f29